src/goto-instrument/cover_location_blocks.h

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************\
+
+Module: Coverage Instrumentation
+
+Author:
+
+\*******************************************************************/
+
+/// \file
+/// Location blocks detection for Coverage Instrumentation
+
+#ifndef CPROVER_GOTO_INSTRUMENT_COVER_LOCATION_BLOCKS_H
+#define CPROVER_GOTO_INSTRUMENT_COVER_LOCATION_BLOCKS_H
+
+#include <goto-programs/goto_model.h>
+#include "cover_basic_blocks.h"
+
+class cover_location_blockst : public cover_blocks_baset
+{
+public:
+  struct block_infot
```

Comment:
Not sure why the data structure from the cover_basic_blockst is insufficient. The bytecode index is part of the source location.